ENIGMAsystem/SHELL/Universal_System/Extensions/BasicGUI/elements.h

```diff
@@ -395,23 +392,23 @@ namespace gui
         if (type == GUI_TYPE::GROUP){
           return data.group;
         }
-        SHOW_ERROR("BasicGUI: Type is not a group! This is going to crash now!\n");
+        DEBUG_MESSAGE("BasicGUI: Type is not a group! This is going to crash now!", MESSAGE_TYPE::M_USER_ERROR);
```

Comment:
These should probably be kept consistent with the other "gonna crash" errors. Thinking a bug is in order to sort this...